pdfarranger/pdfarranger.py

```diff
@@ -727,12 +727,13 @@ def render(self):
 
     def quit_rendering(self):
         """Quit rendering."""
-        if self.rendering_thread:
-            self.rendering_thread.quit = True
-            # If thread is busy with page.render(cr) it might take some time for thread to quit.
-            # Therefore set a timeout here so app continues to stay responsive.
-            self.rendering_thread.join(timeout=0.15)
-            return self.rendering_thread.is_alive()
+        if not self.rendering_thread:
```

Comment:
Should this be `if self.rendering_thread is None:`

Reply:
My version was logically equal to the previous version but yours is even better, I agree.